com.unity.cinemachine/Editor/Utility/InspectorUtility.cs

```diff
@@ -635,37 +671,26 @@ public CompactPropertyField(SerializedProperty property, string label, float min
             }
         }
 
-        public static void AddPropertyDragger(this Label label, SerializedProperty p, VisualElement field)
-        {
-            if (p.propertyType == SerializedPropertyType.Float 
-                || p.propertyType == SerializedPropertyType.Integer)
-            {
-                label.AddToClassList("unity-base-field__label--with-dragger");
-                label.OnInitialGeometry(() =>
-                {
-                    if (p.propertyType == SerializedPropertyType.Float)
-                        new FieldMouseDragger<float>(field.Q<FloatField>()).SetDragZone(label);
-                    else if (p.propertyType == SerializedPropertyType.Integer)
-                        new FieldMouseDragger<int>(field.Q<IntegerField>()).SetDragZone(label);
-                });
-            }
-        }
-
-        public static LeftRightContainer CreatePropertyRow(
+        public static LabeledRow PropertyRow(
             SerializedProperty property, out VisualElement propertyField)
         {
-            var row = new LeftRightContainer();
-
-            var label = row.Left.AddChild(new Label(property.displayName) 
-                { tooltip = property.tooltip, style = { alignSelf = Align.Center, flexGrow = 1 }});
-            propertyField = row.Right.AddChild(new PropertyField(property, "") 
-                { tooltip = property.tooltip, style = { flexGrow = 1, flexBasis = 0 }});
-            AddPropertyDragger(label, property, propertyField);
+            var row = new LabeledRow(property.displayName, property.tooltip);
+            var field = propertyField = row.Contents.AddChild(new PropertyField(property, "")
+                { style = { flexGrow = 1, flexBasis = SingleLineHeight }});
+            AddPropertyDragger(row.Label, property, propertyField);
 
+            // Kill any left margin that gets inserted into the property field
+            field.OnInitialGeometry(() => 
+            {
+                var children = field.Children().GetEnumerator();
+                if (children.MoveNext())
+                    children.Current.style.marginLeft = 0;
+                children.Dispose();
+            });
```

Comment:
Just a note. :)

There are 3 ways to dispose.
1. Dispose manually
```
field.OnInitialGeometry(() => 
{
    var children = field.Children().GetEnumerator();
    if (children.MoveNext())
        children.Current.style.marginLeft = 0;
    children.Dispose(); // call directly
});
```

2. Use variable scope
```
field.OnInitialGeometry(() => 
{
    using var children = field.Children().GetEnumerator(); // this automatically disposes children when children goes out of scope
    if (children.MoveNext())
        children.Current.style.marginLeft = 0;
});
```

3. Define scope
```
field.OnInitialGeometry(() => 
{
    using (var children = field.Children().GetEnumerator()) {
        if (children.MoveNext())
            children.Current.style.marginLeft = 0;
    }
});
```

Reply:
TIL :)